any_parser/any_parser.py

```diff
@@ -137,6 +136,62 @@ def extract(
         else:
             return f"Error: {response.status_code} {response.text}", None
 
+    def extract_json(
+        self,
+        file_path: str,
+        extract_instruction: Dict,
+    ) -> Tuple[str, str]:
+        """Extract json in real-time.
```

Comment:
This looks like a GenAI low quality dosstring, you should not just trust GenAI auto completely output, but given model models about extract what json based on what?

Reply:
Updated with latest commit